aptos/contracts/sources/Governance.move

```diff
@@ -49,19 +51,13 @@ module Wormhole::Governance {
         let action = Deserialize::deserialize_u8(&mut cur);
         assert!(action == 0x02, E_INVALID_ACTION);
 
+        let chain = Deserialize::deserialize_u16(&mut cur);
+        assert!(chain == u16::from_u64(0x00), E_INVALID_TARGET);
+
         let new_index = Deserialize::deserialize_u32(&mut cur);
         let guardian_len = Deserialize::deserialize_u8(&mut cur);
 
-        // TODO - the following assert when we can compare U16 types using (<, >, =)
-        //assert!(guardian_len < 19, E_WRONG_GUARDIAN_LEN);
-
-        while ({
-            spec {
```

Comment:
likewise, removed this spec here. We should look into adding invariants though and enabling `move-prover` in CI, but these particular invariants were not that meaningful